src/storage/data_table.cpp

```diff
@@ -437,7 +437,78 @@ static void VerifyCheckConstraint(TableCatalogEntry &table, Expression &expr, Da
 	}
 }
 
-void DataTable::VerifyAppendConstraints(TableCatalogEntry &table, DataChunk &chunk) {
+static TableIndexList &GetForeignKeyConstraintIndicesAndChunk(const string &schema, const string &table_name,
+                                                              const vector<idx_t> &src_keys,
+                                                              const vector<idx_t> &dst_keys, ClientContext &context,
+                                                              DataChunk &src_chunk, DataChunk &dst_chunk) {
+	auto table_entry_ptr = Catalog::GetCatalog(context).GetEntry<TableCatalogEntry>(context, schema, table_name);
+	if (table_entry_ptr == nullptr) {
+		throw ParserException("Can't find table \"%s\" in foreign key constraint", table_name);
```

Comment:
Can we turn this into an `InternalException`? This should never happen, right?

Reply:
Okay, I will fix it